File: CSharpSDKLib - 4.0/Models/Admin/LoginModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace DMWebLib.Models.Admin
{
    public class LoginModel
    {
        //public LoginModel()
        //{
        //    Identity = new IdentityModel();
        //}
        public IdentityModel Identity { get; set; }
        public string IpAddress { get; set; }
        public DateTime? TimeStamp { get; set; }
    }
}

File: Java/src/com/datamotion/Models/Admin/LoginModel.java
package com.datamotion.Models.Admin;

import com.fasterxml.jackson.annotation.JsonProperty;

public class LoginModel {
	@JsonProperty("Identity")
	private IdentityModel identity;
	
	@JsonProperty("TimeStamp")
	private String currentTimestamp;
	
	@JsonProperty("IpAddress")
	private String ipAddress;
	
	public IdentityModel getIdentity()
	{
		return identity;
	}
	
	public void setIdentity(IdentityModel identity)
	{
		this.identity = identity;
	}
	
	public String getCurrentTimestamp()
	{
		return currentTimestamp;
	}
	
	public void setCurrentTimestamp(String currentTimestamp)
	{
		this.currentTimestamp = currentTimestamp;
	}
	
	public String getIpAddress()
	{
		return ipAddress;
	}
	
	public void setIpAddress(String ipAddress)
	{
		this.ipAddress = ipAddress;
	}
}

